document_page_access_group/views/document_page.xml

```diff
@@ -13,7 +14,12 @@
                     string="Security"
                     groups="document_page.group_document_manager"
                 >
-                    <field name="groups_id" />
+                    <group name="groups" string="Groups" invisible="user_ids">
+                        <field name="groups_id" nolabel="1" colspan="2" />
+                    </group>
+                    <group name="users" string="Users" invisible="groups_id">
+                        <field name="user_ids" widget="many2many_tags" colspan="2" />
```

Comment:
Why `colspan="2"` for a m2m_tags?

Reply:
You are right, without the colspan it displays a little better (taking advantage of all the space).